_maps/RandomRuins/SpaceRuins/oldcodeops.dmm

```diff
@@ -321,15 +316,13 @@
 /turf/open/floor/oldshuttle,
 /area/ruin/powered)
 "Xm" = (
-/obj/structure/chair/old{
-	dir = 4
+/obj/item/paper/crumpled/awaymissions/moonoutpost19/hastey_note{
+	info = "<i>Proceed with caution. This place is better than before, but still dangerous."
```

Comment:
let's not

Reply:
im between the issue of the fact that players are obviously going to metaknowledge and avoid this ruin anyways or walk in and die so this was my best way of telling players "hey, this ruin got changed youll only get mulched slightly less now"

Reply:
This was removed in the following commit.